hanging_threads.py

```diff
@@ -87,7 +86,7 @@ def monitor(seconds_frozen, tests_per_second):
                 log_died_thread(thread_id)
 
         # Process live threads.
-        time.sleep(1. / tests_per_second)
+        time.sleep(test_interval/1000)
```

Comment:
This is ok for Python 3 but for Python 2 it will always be 0. Please make it `1000.`